actions/class.RestQtiTests.php

```diff
@@ -117,9 +126,9 @@ public function import()
                 }
                 return $this->returnSuccess($data);
             } else {
-                throw new \common_exception_RestApi($report->getMessage());
+                throw new common_exception_RestApi($report->getMessage());
             }
-        } catch (\common_exception_RestApi $e) {
+        } catch (common_exception_RestApi $e) {
             return $this->returnFailure($e);
```

Comment:
@shpran would it be asking too much if we could make sure that the exceptions caught here have logs with: 

- Exception class name that was triggered
- Exception message
- Exception trace

Maybe we already have something on `returnFailure` implementation?

I am asking, cause generally this logs with only exception message do not help at all as we have no idea where it happened. This is a critical process for the whole ingestion process, so I think it is useful to have this logs

Wdyt?

Reply:
This one doesn't relate to my changes (just fixed FQN), but sure I will improve these logs 😎 
Thanks for suggestion

Reply:
@shpran I know this is not part of your changes, but today is a bit of a pain to track errors when we are generating translations. Could you please add logs for that?

Reply:
I checked `returnFailuer` implementation and found, that we use Advanced Logger feature there, so message, file, line and trace should be logged:
![image](https://github.com/user-attachments/assets/0df16c5d-baf4-474c-9bc1-d3b80950762b)